### PR TITLE
Follow-up edits to PR#11764

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/11764

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@mrsiano @jeremyeder I made some minor style edits as a follow-up to the original PR. PTAL.

File: scaling_performance/scaling_cluster_monitoring.adoc [hunk 1]
```diff
@@ -14,27 +14,29 @@ toc::[]
...
-nodes].
-* Use at least three OCS nodes (with NVMe drives) link:https://www.redhat.com/en/technologies/cloud-computing/openshift-container-storage[*openshift-container-storage*].
+
+* Use at least three xref:../admin_guide/manage_nodes.adoc#infrastructure-nodes[infrastructure (infra)] nodes.
+* Use at least three
+link:https://www.redhat.com/en/technologies/cloud-computing/openshift-container-storage[*openshift-container-storage*]
+nodes with non-volatile memory express (NVMe) drives.
 * Use persistent storage when configuring *openshift-monitoring*. Set
 `USE_PERSISTENT_STORAGE=true`.
-* Use GlusterFS as storage on top of non-volatile memory express (NVMe) drives link:https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_storage_glusterfs[*openshift_storage_glusterfs*].
```
Comment (mrsiano) [inline scaling_performance/scaling_cluster_monitoring.adoc]: we may want to keep that link ?!

Comment (ahardin-rh) [reply scaling_performance/scaling_cluster_monitoring.adoc]: Fixed!

**Pull request merged**